dropwizard-bom/pom.xml

```diff
@@ -53,6 +53,11 @@
                 <artifactId>guava</artifactId>
                 <version>${guava.version}</version>
             </dependency>
+            <dependency>
+                <groupId>net.bytebuddy</groupId>
+                <artifactId>byte-buddy</artifactId>
```

Comment:
Is the usage of bytebuddy introducing possible fragility in relation to the transitive dependency on asm which generally requires changes based on each new version of java. Which is now occurring at a more rapid pace.

Reply:
That's a valid question -- if there's another (safer) way to dynamically subclass the SpecificBinder, or to register it in such a way that multiple instances of it actually get bound by hk2, I'd love suggestions on that.

Reply:
Also note that mockito-core and hibernate-core already both pull in byte-buddy as a dependency (mockito only in test scope though)

Reply:
hibernate-core is only brought in with dropwizard-hibernate, so this would still be a new dependency to non-hibernate orm users.

You may be able to accomplish the [same thing using Javassist](https://stackoverflow.com/a/17282014/433785), which is already a dependency of dropwizard-core.